src/unstable/components/Form/widgets/TabbedBox.tsx

```diff
@@ -0,0 +1,63 @@
+import * as _ from 'lodash';
+import * as React from 'react';
+import { Box, Button, Flex, Txt } from '../../../../';
+
+interface TabbedBoxProps {
+	tabs: string[];
+	help: JSX.Element;
+	children: JSX.Element[];
+}
+
+interface TabbedBoxState {
+	show: number;
+}
+
+export class TabbedBox extends React.Component<TabbedBoxProps, TabbedBoxState> {
+	constructor(props: TabbedBoxProps) {
+		super(props);
+
+		this.state = {
+			show: 0,
+		};
+	}
+
+	public showTab = (e: React.MouseEvent<HTMLElement>) => {
+		const index = parseInt(e.currentTarget.dataset.index || '', 10);
+
+		this.setState({ show: index });
+	};
+
+	public render() {
+		return (
+			<Box>
+				<Flex justify="space-between" mb={1}>
+					<Flex>
+						{_.map(this.props.tabs, (tab, index: number) => {
+							return (
+								<Button
+									key={index}
+									plaintext
+									data-index={index}
+									mr={3}
+									onClick={this.showTab}
+								>
+									<Txt bold={index === this.state.show}>{tab}</Txt>
+								</Button>
+							);
+						})}
+					</Flex>
+
+					{this.props.help}
+				</Flex>
+
+				{_.map(this.props.children, (child, index) => {
+					if (this.state.show !== index) {
```

Comment:
How about using `_.filter` or `_.find`?

Reply:
I can sidestep any looping and just use an index accessor here